### PR TITLE
Add Saudi Arabia Map

### PR DESCRIPTION
**Pull request by mh-mazen**

### CODE REVIEW

File: packages/saudi-arabia/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "description": "Map of Saudi Arabia",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/VictorCazanave/svg-maps/tree/master/packages/saudi-arabia"
+  },
+  "keywords": [
+    "svg",
+    "map",
+    "new zealand"
```
Comment (VictorCazanave) [inline packages/saudi-arabia/package.json]: Could you replace "new zealand" by "saudi arabia" please?

File: packages/saudi-arabia/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  ],
+  "author": "Victor Cazanave <victor.cazanave@gmail.com>",
```
Comment (VictorCazanave) [inline packages/saudi-arabia/package.json]: You can use your name  😄

File: packages/saudi-arabia/saudi-arabia.svg [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+     title="ar-riyadh"
```
Comment (VictorCazanave) [inline packages/saudi-arabia/saudi-arabia.svg]: Could you replace `title` by `name` attribute please?

File: packages/saudi-arabia/saudi-arabia.svg [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+     id="ard" />
```
Comment (VictorCazanave) [inline packages/saudi-arabia/saudi-arabia.svg]: Could you use more explicit ids please? For example: `ar-riyadh`, `makkah`...

File: packages/saudi-arabia/saudi-arabia.svg [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+     title="makkah"
```
Comment (VictorCazanave) [inline packages/saudi-arabia/saudi-arabia.svg]: Could you use the same names as in the `README.md` file (Riyadh, Makkah...) please?

File: packages/saudi-arabia/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+This map is based on the work of [MapSVG](https://mapsvg.com).
+
+The original map is available [here](https://mapsvg.com/maps/saudi-arabia) and free to be used in any custom project.
+
+### Changes
+
+* Remove unnecessary attributes
+* Replace title by name attributes
+* Add viewBox
+* Rename ids
```
Comment (VictorCazanave) [inline packages/saudi-arabia/README.md]: Could you list only the real changes please?

File: packages/saudi-arabia/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
+# sa Saudi Arabia
+
```
Comment (VictorCazanave) [inline packages/saudi-arabia/README.md]: Could you [add this country in the main `README.md` file](https://github.com/VictorCazanave/svg-maps/blob/master/CONTRIBUTING.md#documentation) please?

**Comment by mh-mazen** [comment]
I Fixed the issues as requested , if there is any further mistakes by me please let me know

**Comment by VictorCazanave** [comment]
Thanks again!

@all-contributors please add @mh-mazen for code

**Pull request merged**